### PR TITLE
Fabo/New transaction design

### PR DESCRIPTION
**Pull request by faboweb**
image

Implementation @jbibla

### CODE REVIEW

**Comment by jbibla** [comment]
wow! amazing @faboweb ! i have a few more updates coming in today and i have a couple recommendations. 

giphy-1

**Comment by jbibla** [comment]
hey @faboweb - here are some design updates: https://sketch.cloud/s/xKnxA/all/voyager-ui/transaction-history

this design should make for some simpler css. please notice that the font size has changed a little bit, how the use of color has changed, and the use of grey vs. white.

**Comment by jbibla** [comment]
lookin' good! here are some css hints: 

the height of the tx-bar is 72px.

css hints

File: src/components/TmBlock/TmBlock.vue [hunk 1]
```diff
@@ -81,14 +82,30 @@ export default {
...
+                    outputs: null
+                  }
+                }
+              ]
             }
           }
         }
       ]
     },
+    candidates: {
```
Comment (jbibla) [inline src/components/TmBlock/TmBlock.vue]: any reason we're using candidates as opposed to validators?

**Comment by faboweb** [comment]
@jbibla Would it be OK if you edit the css yourself? All the structure should be there.

File: src/components/TmBlock/TmBlock.vue (hunk 1 above)
Comment (faboweb) [reply src/components/TmBlock/TmBlock.vue]: I am still making the differentiation between validating candidates and not validating candidates. Shouldn't I?

Comment (jbibla) [reply src/components/TmBlock/TmBlock.vue]: not sure. at some point we should probably sync with the SDK folks about all the words.

Comment (faboweb) [reply src/components/TmBlock/TmBlock.vue]: Ok, changing it.

**Comment by faboweb** [comment]
@jbibla what's the status on this? do you need help?

**Comment by jbibla** [comment]
@faboweb i will work on this today

**Comment by jbibla** [comment]
screen shot 2018-08-20 at 3 09 40 pm

**Comment by jbibla** [comment]
@nylira do you know how to use `SF Pro Text` and `SF Display` in CSS?

**Comment by nylira** [comment]
@jbibla we generally use this in Stylus for the `font-family` in CSS:
`sans = -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif`

Almost all Apple systems will use San Francisco with the `-apple-system` variable.

**Comment by nylira** [comment]
This looks amazing - great job collaborating on this @jbibla and @faboweb!

**Pull request merged**